crystal_toolkit/renderables/structuregraph.py

```diff
@@ -89,8 +89,35 @@ def get_structure_graph_scene(
     legend: Legend | None = None,
     group_by_site_property: str | None = None,
     bond_radius: float = 0.1,
+    site_get_scene_kwargs: dict | None = None,
 ) -> Scene:
-
+    """Returns a Scene containing a representation of the StructureGraph.
```

Comment:
We should prob decide on a consistent doc style format. I think most of CT uses `google` convention:

```
"""Static method for getting plotly figure from a Pourbaix diagram.

Args:
    pourbaix_diagram (PourbaixDiagram): Pourbaix diagram to plot
    heatmap_entry (PourbaixEntry): id for the heatmap generation
    show_water_lines (bool): if True, show region of water stability

Returns:
    (dict) figure layout
"""

Reply:
I was trying to figure this out by looking around the code base and wasn't sure.
If we are set on google then I can update.